src/careamics/config/vae_algorithm_model.py

```diff
@@ -94,23 +101,63 @@ def algorithm_cross_validation(self: Self) -> Self:
                 raise ValueError(
                     f"Algorithm {self.algorithm} only supports loss `musplit`."
                 )
-            if self.model.predict_logvar != "pixelwise":
-                raise ValueError(
-                    "Algorithm `musplit` only supports `predict_logvar` as `pixelwise`."
-                )
-        # TODO add more checks
 
         if self.algorithm == SupportedAlgorithm.DENOISPLIT:
-            if self.loss == SupportedLoss.DENOISPLIT:
+            # TODO: check that noise model and relative likelihood model are well-def
+            if self.loss not in [
+                SupportedLoss.DENOISPLIT,
+                SupportedLoss.DENOISPLIT_MUSPLIT,
+            ]:
                 raise ValueError(
-                    f"Algorithm {self.algorithm} only supports loss `denoisplit`."
+                    f"Algorithm {self.algorithm} only supports loss `denoisplit` "
+                    "or `denoisplit_musplit."
                 )
-            if self.model.predict_logvar is not None:
+            if (
+                self.loss == SupportedLoss.DENOISPLIT
+                and self.model.predict_logvar is not None
+            ):
                 raise ValueError(
-                    "Algorithm `denoisplit` only supports `predict_logvar` as `None`."
+                    "Algorithm `denoisplit` with loss `denoisplit` only supports "
+                    "`predict_logvar` as `None`."
                 )
 
-        # TODO: what if algorithm is not musplit or denoisplit
+        # TODO: what if algorithm is not musplit or denoisplit (HDN?)
```

Comment:
HDN has denoiSplit's loss: noise model enabled with log_var =None

Reply:
I see. Anyway, we'll live HDN for later refactoring, as it most likely require different configs and different naming